web3/_utils/rpc_abi.py

```diff
@@ -52,6 +52,7 @@
     'eth_sendTransaction': TRANSACTION_PARAMS_ABIS,
     'eth_signTransaction': TRANSACTION_PARAMS_ABIS,
     'eth_sign': ['address', 'bytes'],
+    'eth_signTypedData': ['address', None],
```

Comment:
I think a comment got dropped here.  I *think* validation at this level makes sense, but that is really just to ensure that the user gets a good error message if they pass in invalid data.